crates/bevy_ecs/src/schedule/graph/graph_map.rs

```diff
@@ -20,13 +20,13 @@ use Direction::{Incoming, Outgoing};
 ///
 /// For example, an edge between *1* and *2* is equivalent to an edge between
 /// *2* and *1*.
-pub type UnGraph<S = BuildHasherDefault<AHasher>> = Graph<false, S>;
```

Comment:
I'm not sure if the choice to use `BuildHasherDefault<AHasher>` here was heavily motivated, but it felt prudent to use the same default as everywhere else, in particular, using the chosen fixed hasher.

Reply:
Can confirm I just chose a hasher without particular concern. Consistency is a good reason to change it!